RELEASING.md

```diff
@@ -1,9 +1,62 @@
+## Install Github CLI
+
+Install to use the command lines below.
+
+Alternatively can use the github website but ensure you follow the all the --labels --titles etc in the `gh` command lines below.
+
+https://github.com/cli/cli#installation
+
+For rpi:
+
+```bash
+sudo apt update
+sudo apt install gh
+```
+
+## Install github_changelog_generator
+
+If not already installed
+
+github_changelog_generator requires Ruby to be installed.
+Instructions vary based on OS. For rpi:
+
+```bash
+sudo apt-get install rbenv
+sudo apt-get install ruby-dev
+sudo gem install github_changelog_generator
+```
+
+## Get a githubtoken
+
+Or refresh an existing token
+
+This allows the generator to access github.com
+
+visit https://github.com/github-changelog-generator/github-changelog-generator#github-token
+
+```bash
+export CHANGELOG_GITHUB_TOKEN=token
+```
+
 ## Set release information
 
+Replace 0.7.0.dev2 with intended new version number
+
+0.3.5 should always be the previous release as it's the last pyhs100 release in HISTORY.md which is the changelog prior to github release notes.
+
+```bash
+export NEW_RELEASE=0.7.0.dev2
```

Comment:
```suggestion
export NEW_RELEASE=0.
```

Reply:
Maybe
```bash
export NEW_RELEASE=x.x.x.devx
```

Reply:
Also fine, even when it's quite obvious that this is something one should fill.

Reply:
Done